src/app/user/user-page/user-page.component.css

```diff
@@ -26,6 +26,12 @@
     font-family: Arial, Helvetica, sans-serif;
     margin-top: 10px;
 }
+.profile-info h4{
```

Comment:
There is no space between `h4` and opening brace `{` here, however `.buttons {` class selector a few lines down does have that space. Please _make all styles consistent only in this file_ for now.
Please create a new task to fix formatting issues in all `*.css` files and share the new layout/formatting rules with the whole team.